packages/salesforcedx-apex-replay-debugger/src/core/heapDumpService.ts

```diff
@@ -170,7 +224,7 @@ export class HeapDumpService {
     referenceToRevisit: Map<string, ApexVariableContainer>,
     variableTypes: Map<string, string>
   ): void {
-    if (heapDumpExtentValue.value) {
+    if (typeof heapDumpExtentValue.value !== 'undefined') {
```

Comment:
The reason for this change was that the original if statement was checking heapDumpExtentValue.value and if that value was a boolean with a value of false it would fail the if statement. I'd run into this processing the Trigger context boolean values. There's another if statement that had to be changed below.

Reply:
✅